<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="infoLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="fromLayout">
        <item>
         <widget class="QLabel" name="fromLbl">
          <property name="text">
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fromAltEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>Altitude</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fromEdit">
          <property name="text">
           <string>Street</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="toLayout">
        <item>
         <widget class="QLabel" name="toLbl">
          <property name="text">
           <string>To:   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="toAltEdit">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>Altitude</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="toEdit">
          <property name="text">
           <string>Street</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="goButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="distLayout">
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="distLbl">
          <property name="text">
           <string>Distance in meters:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="distLcd"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fuelLayout">
        <item>
         <widget class="QLabel" name="fuelLbl">
          <property name="text">
           <string>Wasted fuel in liters:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="fuelLcd"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="moneyLayout">
        <item>
         <widget class="QLabel" name="moneyLbl">
          <property name="text">
           <string>Wasted $ in AR$:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="moneyLcd"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mapLayout">
      <item>
       <widget class="CGraphicsZoom" name="graphicsView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionDetailed_path">
   <property name="text">
    <string>Detailed path</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CGraphicsZoom</class>
   <extends>QGraphicsView</extends>
   <header>cgraphicszoom.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
